beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md

```diff
@@ -28,7 +28,7 @@ This feature allows you to write a function to validate events in your Project G
 | `uuid` | String | Universally unique identifier for the event. |
 | `timestamp` | Number | Timestamp from the device when the event was sent from the SDK. |
 | `eventNumber` | Number | Used to order when the event was sent. This key is useful when events have the same timestamp. |
-| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.griffon\). |
+| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.assurance\). |
```

Comment:
For now, we aren't going to change our vendor name to com.adobe.assurance. So, let's leave this example as com.adobe.griffon for now so it matches what people will see.

Reply:
OK, I reverted this change.